tox.ini

```diff
@@ -19,8 +19,16 @@ commands =
 select = F401,F402,F821,F823,F841,F811,E265,E266,F541,W605,E722,F822,F523,W291,F405
 exclude = awx/ui/node_modules,awx/ui/node_modules,env,awx_collection_build
 
+[testenv:pip-compile-docs]
+description = Compile docs build requirements
+allowlist_externals = pip-compile
+commands =
+  pip-compile --allow-unsafe --output-file=docs/docsite/requirements.txt --strip-extras docs/docsite/requirements.in
```

Comment:
Jag recently added a config file support: https://pip-tools.readthedocs.io/en/latest/#configuration — why not put most of the args there?

Reply:
I've updated to use `pyproject.toml` instead of introducing a separate `pip-tools.toml`

Reply:
I'm not a fan of dumping different tool configs into the same file. But since this is not a mass-maintained project, this probably doesn't need those scaling considerations...